src/main/java/io/curity/identityserver/plugin/criipto/authentication/RedirectUriUtil.java

```diff
@@ -0,0 +1,28 @@
+package io.curity.identityserver.plugin.criipto.authentication;
+
+import se.curity.identityserver.sdk.errors.ErrorCode;
+import se.curity.identityserver.sdk.service.ExceptionFactory;
+import se.curity.identityserver.sdk.service.authentication.AuthenticatorInformationProvider;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static io.curity.identityserver.plugin.criipto.descriptor.CriiptoAuthenticatorPluginDescriptor.CALLBACK;
+
+public class RedirectUriUtil
```

Comment:
should be final and have a private ctor; also IINM it can be package private.

Reply:
True. I will change that also in the other repositories.